api/types/types.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// ContainerCreateResponse contains the information returned to a client on the
+// creation of a new container.
+type ContainerCreateResponse struct {
+	// ID is the id of the created container.
+	ID string `json:"Id"`
```

Comment:
Hmm, you should be sure by test, not by some string, which also easy to misspell. We had such "100% fact" already in `devicemapper` code with `json: "Id"`.
But I'm okay if you want save them.